### PR TITLE
Backlog Security BugFix

### PR DESCRIPTION
**Pull request by takashi-uesaka**

### CODE REVIEW

**Comment by kevinchalet** [comment]
Looks like your PR includes an unwanted commit that causes a conflict.
Note that you'll also have to remove `httpClient.DefaultRequestHeaders.Authorization = null;` :sake: 

To get rid of the weird commit, try rebasing from this repository by running `git fetch upstream` + `git rebase upstream/master`. Then, try a forced push: `git push -f`

**Comment by jerriep** [comment]
@takashi-uesaka Can you please resolve the conflicts on this one please?

**Comment by takashi-uesaka** [comment]
Sorry for my late work. I could get back to this work after few days.

**Comment by jerriep** [comment]
No problem. Whenever you're ready :)

**Comment by kevinchalet** [comment]
> No problem. Whenever you're ready :)

Actually, that would be a very nice opportunity to try cherry picking :smile:

**Comment by takashi-uesaka** [comment]
Thank you for waiting. It was done.
I removed httpClient.DefaultRequestHeaders.Authorization = null; and did git fetch upstream + git rebase upstream/master as you said.
please check it out.

File: Owin.Security.Providers/Backlog/BacklogAuthenticationHandler.cs [hunk 1]
```diff
@@ -72,9 +73,12 @@ public BacklogAuthenticationHandler(HttpClient httpClient, ILogger logger)
...
 
                 // Get token
-                httpClient.DefaultRequestHeaders.Authorization = null;
-                HttpResponseMessage tokenResponse =
-                    await httpClient.PostAsync(Options.TokenEndpoint, new FormUrlEncodedContent(body));
+                var tokenRequest = new HttpRequestMessage(HttpMethod.Post, Options.TokenEndpoint);
+                tokenRequest.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
+                tokenRequest.Content = new FormUrlEncodedContent(body);
+
+                HttpResponseMessage tokenResponse = await httpClient.SendAsync(tokenRequest);
```
Comment (kevinchalet) [inline Owin.Security.Providers/Backlog/BacklogAuthenticationHandler.cs]: nit: you could use `await httpClient.SendAsync(tokenRequest, Request.CallCancelled);`, but that's a detail.

**Comment by kevinchalet** [comment]
:shipit:

**Comment by takashi-uesaka** [comment]
I added the Request.CallCancelled.
This time, I squashed the last commit and did rebase upstream/master.

**Pull request merged**